include/LMS1xx/colaa.h

```diff
@@ -0,0 +1,268 @@
+#ifndef COLAA_H
```

Comment:
we should also add license headers. Copy from other file and add yourself as author. (I know, this also goes for the MRS files ;-) )